README.md

```diff
@@ -7,6 +7,8 @@ This project very much under active development, and is currently not as reusabl
 
 ## Configuring for Development
 
+Make sure an instance of elasticsearch is available at `http://elasticsearch.local:9200/`. If you are running theonion/onion-services you can add `elasticsearch.local` to your `/etc/hosts` line that points at that docker dev machine.
+
```

Comment:
Hmmm you'll have to show me how you forwarded that port -- that might cleanup our need to use /etc/hosts at all for our *.local dev sites too.